Doc/library/html.parser.rst

```diff
@@ -126,16 +126,16 @@ implementations do nothing (except for :meth:`~HTMLParser.handle_startendtag`):
 
 .. method:: HTMLParser.handle_starttag(tag, attrs)
 
-   This method is called to handle the start of a tag (e.g. ``<div id="main">``).
+   This method is called to handle the start tag of an element (e.g. ``<div id="main">``).
```

Comment:
IMO this change is correct and useful, but the other changes are rewordings that do not make the text more exact or useful.

Reply:
I have just removed the other consistency changes and kept this fix.